README.md

```diff
@@ -122,6 +122,7 @@ To ease development, you can also use `npm run compile-continuously` to have you
 
 > In this case, you could probably also make use of [`live-server`](https://www.npmjs.com/package/live-server).
 
+Or you can even run `npm run dev` for automatic page reloading on change at the url: `http://localhost:8080/webpack-dev-server/cout-embauche-widget`
```

Comment:
The logged url which does not end with `cout-embauche-widget` only opens index.html, which points to the dist/ widget, and will not be automatically reloaded. 
`[output url] /cout-embauche-widget` then ?